### PR TITLE
started translating forms

### PR DESCRIPTION
**Pull request by nahidmbstu**
Very happy to see that React doc is being translated to bangla. I am senior developer from Dhaka , Bangladesh working on React , React Native. 

I have contributed few translation and will be providing more in future. Thanks

### CODE REVIEW

File: content/docs/forms.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 id: forms
-title: Forms
+title: ফরম
```
Comment (shakib609) [inline content/docs/forms.md]: ফর্ম

File: content/docs/forms.md [hunk 2]
```diff
@@ -9,19 +9,20 @@ redirect_from:
   - "docs/forms-zh-CN.html"
 ---
 
-HTML form elements work a little bit differently from other DOM elements in React, because form elements naturally keep some internal state. For example, this form in plain HTML accepts a single name:
+
+এইচটিএমএল form উপাদানগুলি React এ  অন্যান্য DOM উপাদানের থেকে আলাদা আলাদাভাবে কাজ করে , কারণ form উপাদান এর  স্বাভাবিকভাবে কিছু অভ্যন্তরীণ state আছে। উদাহরণস্বরূপ, এই  HTML ফর্মটি একটি একক নাম  গ্রহণ করে :
```
Comment (shakib609) [inline content/docs/forms.md]: Please follow this: [Glossary](https://github.com/reactjs/bn.reactjs.org/wiki/Glossary)

File: content/docs/forms.md [hunk 3]
```diff
@@ -9,19 +9,20 @@ redirect_from:
...
 
 ```html
 <form>
   <label>
-    Name:
+    নাম:
```
Comment (shakib609) [inline content/docs/forms.md]: I don't think codes should be changed. What do you think @nutboltu and @sazzadsazib  vai?

Comment (Twaha-Rahman) [reply content/docs/forms.md]: @shakib609 I agree, changing the code will create confusion. But comments should be translated

File: content/docs/forms.md (hunk 1 above)
Comment (ImranMH) [reply content/docs/forms.md]: What about keeping it as "Forms". Form is an important concept also forms is the plural of form.

File: content/docs/forms.md [hunk 4]
```diff
@@ -9,19 +9,20 @@ redirect_from:
...
     <input type="text" name="name" />
   </label>
   <input type="submit" value="Submit" />
 </form>
 ```
 
-This form has the default HTML form behavior of browsing to a new page when the user submits the form. If you want this behavior in React, it just works. But in most cases, it's convenient to have a JavaScript function that handles the submission of the form and has access to the data that the user entered into the form. The standard way to achieve this is with a technique called "controlled components".
+এই ফর্মটির ডিফল্ট HTML ফর্ম আচরণ হল , ব্যবহারকারীর ফর্মটি submit করলে এই ফর্মটি একটি নতুন পৃষ্ঠায় নিয়ে যায় । আপনি যদি এই আচরণ চান , React এ এটা কাজ করে । তবে বেশিরভাগ ক্ষেত্রে, এটি একটি জাভাস্ক্রিপ্ট ফাংশন থাকা সুবিধাজনক যা ফর্মটি জমা দেওয়ার পরিচালনা করে  এবং  ব্যবহারকারীর form ডাটা অ্যাক্সেস থাকে। এটি অর্জনের আদর্শ উপায় "নিয়ন্ত্রণাধীন উপাদান" নামে একটি কৌশল।
```
Comment (ImranMH) [inline content/docs/forms.md]: misleading translation. this portion needs a better translation.
controlled components !== "নিয়ন্ত্রণাধীন উপাদান"

Comment (sazzadsazib) [reply content/docs/forms.md]: I think we should call "controlled components" as ite mentioned as technique

**Comment by nutboltu** [comment]
@nahidmbstu Any update on this PR?

**Comment by nutboltu** [comment]
@nahidmbstu Can we please have a look?

**Comment by Twaha-Rahman** [comment]
The last commit was an year ago...Is this PR still alive?

**Comment by nutboltu** [comment]
@nahidmbstu would you please respond to this PR or we will mark it a stale

**Pull request closed without merging**